@navikt/core/react/src/date/datepicker/types.ts

```diff
@@ -52,8 +53,16 @@ export interface DatePickerDefaultProps
   /**
    * Changes datepicker locale
    * @default "nb" (norsk bokmål)
+   * @deprecated Use `<Provider />`-component
    */
   locale?: "nb" | "nn" | "en";
+  /**
+   * i18n-API for customizing texts and labels.
+   *
+   * **NB: If you need to change the language, use the Provider instead.**
+   * @see [Provider](https://aksel.nav.no/komponenter/core/provider#84d7ea5ec517)
+   */
```

Comment:
What do you think about this instead?
```suggestion
   * **NB: If you need to change the language, use [Provider](https://aksel.nav.no/komponenter/core/provider#84d7ea5ec517) instead.**
   */
```

Reply:
That works 🚀